life_expectancy/country.py

```diff
@@ -0,0 +1,73 @@
+"""List of all countries present in the input dataset'"""
+from enum import Enum
+
+class Country(str, Enum):
+    """List of available countries"""
+    AL = "AL"
+    AM = "AM"
+    AT = "AT"
+    AZ = "AZ"
+    BE = "BE"
+    BG = "BG"
+    BY = "BY"
+    CH = "CH"
+    CY = "CY"
+    CZ = "CZ"
+    DE = "DE"
+    DE_TOT = "DE_TOT"
+    DK = "DK"
+    EA18 = "EA18"
+    EA19 = "EA19"
+    EE = "EE"
+    EEA30_2007 = "EEA30_2007"
```

Comment:
The list of available countries should only be possible real countries. Remove these non-sense values